security-analysis/security-analysis-api/src/main/java/com/powsybl/security/PostContingencyResult.java

```diff
@@ -22,9 +26,19 @@
 
     private final LimitViolationsResult limitViolationsResult;
 
+    Map<String, BranchResult> branchResults;
+
+    Map<String, BusResults> busResults;
```

Comment:
We must provide getters for the new fields. More precisely, in order to not let the user modify anything in the map, we can provide accessors for the collection fo results, or for on result by its ID:
```java
public BusResult getBusResult(String id) { return busResults.get(id); }
public Collection<BusResult> getBusResults()  { return busResults.values(); }
```
Same for branch results